crates/services/tx_status_manager/src/config.rs

```diff
@@ -0,0 +1,22 @@
+use std::time::Duration;
+
+#[derive(Clone, Debug)]
+pub struct Config {
+    /// Maximum of subscriptions to listen to updates of a transaction.
+    pub max_tx_update_subscriptions: usize,
+    /// Maximum transaction time to live.
+    pub max_txs_ttl: Duration,
```

Comment:
Why do we need it? TxPool is responsible for notifying that the transaction was removed(or consensus module in the case of preconfirmation and sequeze out).

Reply:
Or do you want to have a deadline for subscriptions? IF yes, then it should be `subscription_ttl`